### ISSUE TITLE
Armor Stand visibility toggle not available

### ISSUE THREAD

**Issue by c0wg0d** [Wolfst0rm/ArmorStandEditor-Issues#21]
### Expected behavior

Toggle for visibility when right clicking with flint

### Observed/Actual behavior

I don't see this option (the potion in the GUI)

### Steps/models to reproduce

Install ArmorStandEditor-1.18.1-32 with default config

Right click with flint

### Plugin list

PlotSquared
LuckPerms
ArmorStandEditor

### Plugin Version

1.18.1-32

### Server Version

Paper version git-Paper-402 (MC: 1.17.1) (Implementing API version 1.17.1-R0.1-SNAPSHOT) (Git: 1bc077b on ver/1.17.1)

### Other

I added these options in my config:
```
#Allow the option to toggle invisiblity for ItemFrames and ArmorStand
ivisibleItemFrames: true
armorStandVisibility: true
```

Shouldn't the option be available by default?  I also tried adding the permission `asedit.armorstand.invisible` but that didn't let me toggle visibility either.

**Comment by c0wg0d** [comment]
If I give myself the permission `asedit.invisible` then the toggle is available.  Is this intended behavior?  It seems like the permission should not be required if the config is set to true for invisibility.

**Comment by Wolfieheart** [comment]
Already fixed in dev/1.18.1-33 https://github.com/Wolfst0rm/ArmorStandEditor/pull/22/commits/71e40e3faec474834c0763c1bf276368391b59cf
